src/api.js

```diff
@@ -0,0 +1,62 @@
+import axios from "axios";
+import { useEffect, useState } from "react";

```

Comment:
remove unnecessary imports - we don't need react stuff in here

Reply:
done